src/update.js

```diff
@@ -0,0 +1,8 @@
+import { saveToLocal } from './crud.js';
+
+const updateCompleted = (index, arr, status) => {
+  arr[index - 1].completed = status;
+  saveToLocal(arr);
+};
+
+export default updateCompleted;
```

Comment:
you project is descriptive u use the best practices of js html and css
your project is  DRY, KISS and YAGNI